reporter/src/main/kotlin/utils/FreemarkerTemplateProcessor.kt

```diff
@@ -330,6 +330,17 @@ class FreemarkerTemplateProcessor(
         @Suppress("UNUSED") // This function is used in the templates.
         fun filterForUnresolvedVulnerabilities(vulnerabilities: List<Vulnerability>): List<Vulnerability> =
                 vulnerabilities.filter { resolutionProvider.getVulnerabilityResolutionsFor(it).isEmpty() }
+
+        /**
+         * Return a human-readable string that is determined based on the given score and scoring system.
+         */
+        @Suppress("UNUSED") // This function is used in the templates.
+        fun getSeverityString(scoreSystem: String?, score: String) =
```

Comment:
We should use `scoringSystem` instead of `scoreSystem` to align with the variable name in `VulnerabilityReference`.